types/sharp/index.d.ts

```diff
@@ -718,20 +719,446 @@ declare namespace sharp {
+         * - `8` -  Adobe Deflate
+         * - `9` -  JBIG B&W
+         * - `10` -  JBIG Color
+         * - `99` -  JPEG
+         * - `262` -  Kodak 262
+         * - `32766` -  Next
+         * - `32767` -  Sony ARW Compressed
+         * - `32769` -  Packed RAW
+         * - `32770` -  Samsung SRW Compressed
+         * - `32771` -  CCIRLEW
+         * - `32772` -  Samsung SRW Compressed 2
+         * - `32773` -  PackBits
+         * - `32809` -  Thunderscan
+         * - `32867` -  Kodak KDC Compressed
+         * - `32895` -  IT8CTPAD
+         * - `32896` -  IT8LW
+         * - `32897` -  IT8MP
+         * - `32898` -  IT8BL
+         * - `32908` -  PixarFilm
+         * - `32909` -  PixarLog
+         * - `32946` -  Deflate
+         * - `32947` -  DCS
+         * - `33003` -  Aperio JPEG 2000 YCbCr
+         * - `33005` -  Aperio JPEG 2000 RGB
+         * - `34661` -  JBIG
+         * - `34676` -  SGILog
+         * - `34677` -  SGILog24
+         * - `34712` -  JPEG 2000
+         * - `34713` -  Nikon NEF Compressed
+         * - `34715` -  JBIG2 TIFF FX
+         * - `34718` -  Microsoft Document Imaging (MDI) Binary Level Codec
+         * - `34719` -  Microsoft Document Imaging (MDI) Progressive Transform Codec
+         * - `34720` -  Microsoft Document Imaging (MDI) Vector
+         * - `34887` -  ESRI Lerc
+         * - `34892` -  Lossy JPEG
+         * - `34925` -  LZMA2
+         * - `34926` -  Zstd
+         * - `34927` -  WebP
+         * - `34933` -  PNG
+         * - `34934` -  JPEG XR
+         * - `65000` -  Kodak DCR Compressed
+         * - `65535` -  Pentax PEF Compressed
+         */
+        Compression?: number;
+        /**
+         * - `0` - WhiteIsZero
+         * - `1` - BlackIsZero
+         * - `2` - RGB
+         * - `3` - RGB Palette
+         * - `4` - Transparency Mask
+         * - `5` - CMYK
+         * - `6` - YCbCr
+         * - `8` - CIELab
+         * - `9` - ICCLab
+         * - `10` - ITULab
+         * - `32803` - Color Filter Array
+         * - `32844` - Pixar LogL
+         * - `32845` - Pixar LogLuv
+         * - `32892` - Sequential Color Filter
+         * - `34892` - Linear Raw
+         * - `51177` - Depth Map
+         */
+        PhotometricInterpretation?: number;
+
+        /**
+         * - `1` - No dithering or halftoning
+         * - `2` - Ordered dither or halftone
+         * - `3` - Randomized dither
+         */
+        Thresholding?: 1 | 2 | 3;
+
+        DocumentName?: string;
+        ImageDescription?: string;
+        Make?: string;
+        Model?: string;
+        /**
+         * - `1` - Horizontal (normal)
+         * - `2` - Mirror horizontal
+         * - `3` - Rotate 180
+         * - `4` - Mirror vertical
+         * - `5` - Mirror horizontal and rotate 270 CW
+         * - `6` - Rotate 90 CW
+         * - `7` - Mirror horizontal and rotate 90 CW
+         * - `8` - Rotate 270 CW
+         */
+        Orientation?: 1 | 2 | 3 | 4 | 5 | 6 | 7 | 8;
```

Comment:
Shouldn't this type simply be `ImageOrientation` as declared [here](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/53942/files#diff-3a3a3cb51b7f272cca4baf692cde8c1aa7b9265ca2b29e1e9d9ac5f40388bd10R722)?
(e.g. `Orientation?: ImageOrientation`)